src/outro.js

```diff
@@ -1,2 +1,13 @@
+for (var key in MathQuill) (function(key, val) {
+  if (preInterVerMathQuill[key]) return; // already set .noConflict
+  if (typeof val === 'function') {
+    preInterVerMathQuill[key] = function() {
+      insistOnInterVer();
```

Comment:
I'm a little confused about how this will work out in practice. Does this mean that people who don't specify an interfaceVersion will see a console warning for each API call they make?

Is it considered valid to not declare an interface version until you are ready to opt in to version 1? Or is version 1 effectively the same as the pre InterVer behavior, and we're expecting that everyone that gets on "MathQuill 2.0" (if that's what we're calling the dev branch once it's released) should definitely immediately specify an InterVer version.

In any case, I think it would make sense to reduce the noise a little bit, and only do this warning either once, or once per different API method call.